config/locales/en/activerecord/course/assessment/submission.yml

```diff
@@ -12,6 +12,7 @@ en:
         course/assessment/submission:
           experience_points_record:
             inconsistent_user: 'the creator of the submission must be the same as the course user'
+          submission_already_exists: "Submission with ID %{existing_id} already exists"
```

Comment:
@fonglh Small issue, can we change the message so that it's more user friendly ? I think user probably don't understand the id.

How about:
"Looks like you have already created a submission, try again ?", or sth similar.  

@weiqingtoh any suggestions ?